syncer/checkpoint.go

```diff
@@ -649,51 +669,65 @@ func (cp *RemoteCheckPoint) Load(tctx *tcontext.Context, schemaTracker *schema.T
 // LoadMeta implements CheckPoint.LoadMeta
 func (cp *RemoteCheckPoint) LoadMeta() error {
 	var (
-		pos *mysql.Position
-		err error
+		location *binlog.Location
+		err      error
 	)
 	switch cp.cfg.Mode {
 	case config.ModeAll:
 		// NOTE: syncer must continue the syncing follow loader's tail, so we parse mydumper's output
 		// refine when master / slave switching added and checkpoint mechanism refactored
-		pos, err = cp.parseMetaData()
+		location, err = cp.parseMetaData()
 		if err != nil {
 			return err
 		}
 	case config.ModeIncrement:
 		// load meta from task config
 		if cp.cfg.Meta == nil {
 			cp.logCtx.L().Warn("don't set meta in increment task-mode")
+			location1 := binlog.NewLocation(cp.cfg.Flavor)
+			cp.globalPoint = newBinlogPoint(location1, location1, nil, nil)
 			return nil
 		}
-		pos = &mysql.Position{
-			Name: cp.cfg.Meta.BinLogName,
-			Pos:  cp.cfg.Meta.BinLogPos,
+		gset, err := gtid.ParserGTID(cp.cfg.Flavor, cp.cfg.Meta.BinLogGTID)
+		if err != nil {
+			return err
+		}
+		if gset == nil {
```

Comment:
In which situation will `gset` be nil? It sames that `gset` will always have a value.

Reply:
removed in a59f087